### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by dethos**
This pull request removes files that are no longer needed and updates some contents as described on the list of commits.

### CODE REVIEW

File: boxes/templates/boxes/box_list.html [hunk 1]
```diff
@@ -164,64 +156,43 @@ <h3 class="subtitle center-xs">Create New Box</h3>
...
-
-<script src="{% static 'javascripts/box_list.js' %}"></script>
-
-
-<script type="text/javascript" src="{% static 'javascripts/jquery.datetimepicker.min.js' %}"></script>
-<script src="{% static 'javascripts/box_create.js' %}"></script>
-
-
-
+  {{ block.super }}
```
Comment (pjaneiro) [inline boxes/templates/boxes/box_list.html:175]: Are there no duplicate files? @dethos

File: boxes/templates/boxes/box_submit.html [hunk 2]
```diff
@@ -16,8 +16,8 @@
         var popup = document.getElementById('thisPopup');
         popup.classList.toggle('show');
     }
-    
-    
+
+
```
Comment (pjaneiro) [inline boxes/templates/boxes/box_submit.html]: Might as well delete these two lines, no?

**Comment by pjaneiro** [comment]
@dethos Everything seems ok, I just think there a few blank lines that might as well be deleted, but otherwise I think it's ready to ship

File: boxes/templates/boxes/box_list.html (hunk 1 above)
Comment (dethos) [reply boxes/templates/boxes/box_list.html:175]: There shouldn't be

**Comment by dethos** [comment]
@Budgerous rearranged some of the templates and removed the blank lines. If everything is ok, please merge. cc: @jlbribeiro

**Comment by pjaneiro** [comment]
I think it's ready to merge, Let's just wait for @jlbribeiro's opinion

File: boxes/templates/boxes/box_list.html (hunk 1 above)
Comment (jlbribeiro) [reply boxes/templates/boxes/box_list.html:175]: @Budgerous Can you please clarify what you meant with your comment? Does it affect this line in particular?

**Comment by jlbribeiro** [comment]
LGTM, merging :+1:

**Pull request merged**